components/sendgrid/sendgrid.app.js

```diff
@@ -68,5 +72,23 @@ module.exports = {
         retriableStatusCodes,
       );
     },
+    async getWebhookSettings() {
+      const url = this._webhookSettingsUrl();
+      const requestConfig = this._makeRequestConfig();
+      const retriableStatusCodes = new Set([408, 429]);
```

Comment:
nit but if these are always the same maybe just default in `withRetries`?

Reply:
That's a good idea, looks like all API endpoints share these status codes, so we can reuse them.
Will fix!